deps.ffmpeg/60-mbedtls.zsh

```diff
@@ -3,21 +3,27 @@ autoload -Uz log_debug log_error log_info log_status log_output
 ## Dependency Information
 local name='mbedtls'
 local -A versions=(
-  macos 3.2.1
-  linux 3.2.1
-  windows 3.2.1
+  macos 3.4.0
+  linux 3.4.0
+  windows 3.4.0
```

Comment:
Did we determine in the end which version of mbedTLS is safe to use for our needs?

Reply:
I personally tested with 3.4.0 which does work.
Maybe 3.2.1 would work too though.

Reply:
3.4.0 is what we want. mbedTLS fixed their includes on Windows with 3.4.0

Reply:
> mbedTLS fixed their includes on Windows with 3.4.0

Not that I'm aware of. The GitHub Issue cited to me was:
https://github.com/Mbed-TLS/mbedtls/issues/7087

Reply:
@RytoEX I believe the particular header causing us issues was fixed. Looking at the discussion I believe others are struggling with different includes.

https://discord.com/channels/348973006581923840/1052657814264356955/1092873798749016155 is the relevant conversation in development channel.

Reply:
The linked GitHub Issue is about Designated Initializers (a GCC C++ extension not in the C++ standard until C++20) in header files. My replies in that conversation were not refuted directly, so I had no reason to believe that the issue with Designated Initializers was resolved intentionally.

My relevant replies in the conversation that you linked:
> IIRC, the issue is that their headers use Designated Initializers, which work in GCC because of GCC extensions, but do not work in standard C++ in MSVC until C++20, and they aren't comfortable using C++20 as their baseline yet.
> I don't think we tested with 3.4.0 to find out. It wasn't out yet.
> 3.4.0 only came out last week (March 28).
> So we wouldn't have tested it two weeks ago.
> If they accidentally fixed it, neat. They don't seem to be aware of that being the case though, or they would have closed the Issue.

If, in fact, the GitHub Issue cited to me was not the actual problem, which seems to be implied by [your follow-up comment](https://github.com/paullouisageneau/libdatachannel/pull/822#issuecomment-1478892712) on the libdatachannel PR to add mbedTLS support, then it was simply not something I recalled or perhaps not clarified in conversations that I followed closely.

Reply:
@RytoEX My comment is not related to this. The required includes changed. mbedTLS 3.4.0 wasn’t working on all platforms.

I am pretty confident that code mbedTLS stopped usage of designated initializations.

what do we need here to move forward?

@pkviet does what I say match with what you saw?

Reply:
@RytoEX I will fix the Windows builds now.

Reply:
Tested with mbedtls 3.2.1; the datachannel dll worked fine.
There's no need to update mbedtls on windows account.

Reply:
clarification following internal discussion: in my tests, BOTH mbedtls 3.2.1 and 3.4.0 produced libdatachannel dlls which work with the current WHIP PR (using the C API of libdatachannel).